README.md

```diff
@@ -0,0 +1,22 @@
+# Essex microsites
+
+This project was originally installed using the [Composer](https://getcomposer.org/) project template at [LocalGov Drupal Microsites](https://github.com/localgovdrupal/localgov_microsites_project). More [documentation about LGD Microsites can be found here](https://docs.localgovdrupal.org/microsites/).
+
+## Quick start for local development
+
+Use [DDEV](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/).
+
+1. Clone this repo.
+2. Run `ddev start` in the project root.
+3. Run `ddev composer install` to install the project dependencies.
+4. Run `ddev drush si localgov_microsites`.
+
+The controller site is then available at https://ecc-microsites.ddev.site
```

Comment:
Following these instructions gives a microsites site, but without the Essex theme

Reply:
The `localgov_microsites` profile is unaware of the essex theme, and also I noticed the theme is mentioned in another ticket (ECCW-570). What I'll do is have composer swap `web/profiles/contrib/localgov_microsites/localgov_microsites.info.yml` for a modified version in that upcoming PR, if that's OK with you.